.azure/pipelines/jobs/default-build.yml

```diff
@@ -8,7 +8,7 @@
 #   poolName: string
 #       The name of the Azure DevOps agent pool to use.
 #   agentOs: string
-#       Used in templates to define variables which are OS specific. Typically from the set { Windows, Linux, macOS }
+#       Used in templates to define variables which are OS specific. Typically from the set { Windows, Linux, MacOS }
```

Comment:
Why change the case of `macOS`❔ Comparisons aren't case-sensitive.

Reply:
Just for consistency, it was bothering me that some job names started with uppercase and some with lowercase so I did a find-replace.

Reply:
Interesting, I thought `macOS` is more canon than `MacOS` since that's how it's stylized by Apple

Reply:
Didn't know that - I can always change it back, not too big of a deal